pkg/interface/config/webpack.prod.js

```diff
@@ -56,20 +60,22 @@ module.exports = {
     new CleanWebpackPlugin(),
     new webpack.DefinePlugin({
       'process.env.LANDSCAPE_STREAM': JSON.stringify(process.env.LANDSCAPE_STREAM),
-      'process.env.LANDSCAPE_SHORTHASH': JSON.stringify(process.env.LANDSCAPE_SHORTHASH),
+      'process.env.LANDSCAPE_SHORTHASH': JSON.stringify(GIT_DESC),
```

Comment:
Why change this? We pull the hash from the bash env on deploy, what’s this get us?

Reply:
For one, I think using the git description is better, because you can see the version number. For two, I wanted to inject it at build time, instead of using an env var.

Reply:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/20846414/108652036-21bc4a80-7491-11eb-81a2-99770d95298f.png">

👀 this is at least a little preferable to our base hash indicator honestly. @urcades for further contemplation